hkm/templates/hkm/snippets/search_records_grid.html

```diff
@@ -10,24 +10,16 @@
         </svg>
     </a>
     <a href="{% url 'hkm_search_record' %}?image_id={{ record.id }}">
-      {% if user.is_authenticated %}
-        <div class="grid__fav {% if record.is_favorite %}active{% endif %}" data-record-id="{{ record.id }}">
-          <i class="grid__fav-icon">
-            <svg class="svg-icon">
-              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-heart-stroke"></use>
-            </svg>
-          </i>
-        </div>
-      {% endif %}
-      <img class="flex-img" src="{% finna_image record.id w=400 h=400 %}"></img>
+      <img class="flex-img" src="{% finna_image record.id w=400 h=400 %}" alt="{{ record.title }}"></img>
```

Comment:
What's happening here? It seems like you are removing content for authenticated users.

Reply:
For some reason the favourite button was decleared twice, the one I removed don't seem to be used any scenario. I decided to remove it now so it won't cause more confusion in the furure.